sdk/tests/deqp/framework/common/tcuFloat.js

```diff
@@ -596,37 +633,81 @@ tcuFloat.numberToFloat11 = function(value) {
     return tcuFloat.newFloat11(value).bits();
 };
 
-tcuFloat.float11ToNumber = function(float11) {
-    var x = tcuFloat.newDeFloatFromParameters(float11, tcuFloat.description11);
-    return x.getValue();
-};
+tcuFloat.float11ToNumber = (function() {
+    var x = tcuFloat.newDeFloatFromParameters(0, tcuFloat.description11);
+    return function(float11) {
+        x.deFloatParametersNumber(float11);
+        return x.getValue();
+    };
```

Comment:
To clarify: is one of the main optimizations to reuse the same deFloat instance over and over again for these conversions?